### PR TITLE
Add my favorite things .txt

### PR DESCRIPTION
**Pull request by Awounor**
Add a list of thing I like because that was asked.

### CODE REVIEW

File: week3/intro.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 My five favorite things are:
-- 
-- 
-- 
-- 
-- 
+- Sport(Volleyball)
+- New Technology
+- Traveling 
+- Meeting new peoples
```
Comment (Megan-Sullivan) [inline week3/intro.txt]: "peoples" -> "people"

File: week3/intro.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+- Road trips
```
Comment (Megan-Sullivan) [inline week3/intro.txt:6]: "trips" -> "Trips"

**Review by Megan-Sullivan — CHANGES_REQUESTED** [review]

File: week3/intro.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 My five favorite things are:
-- 
-- 
-- 
-- 
-- 
+- Sport(Volleyball)
```
Comment (danjwuest) [inline week3/intro.txt:2]: I think volleyball is a pretty cool sport to play too!

**Review by danjwuest — APPROVED** [review]

File: week3/intro.txt (hunk 3 above)
Comment (Awounor) [reply week3/intro.txt:2]: I couldn't agree more. I even started a coed volleyball club at cc this quarter.

**Review by Awounor — COMMENTED** [review]
Fix vocabulary error

**Pull request merged**